rust/fastsim-core/src/cycle.rs

```diff
@@ -726,6 +726,19 @@ impl SerdeAPI for RustCycle {
             ),
         }
     }
+
+    /// takes an object from a url and saves it in the fastsim data directory in a rust_objects folder
```

Comment:
make sure there is a trailing double space, `  ` on this line so that the following line is actually on the next line when rendered.  You can check rendering by running `cargo doc --no-deps --open`.